pkg/config/config.go

```diff
@@ -250,6 +250,7 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("kubelet_client_key", "")
 
 	config.BindEnvAndSetDefault("kubelet_wait_on_missing_container", 0)
+	config.BindEnvAndSetDefault("kubelet_cache_pods_duration", 3) // Polling frequency in seconds of the agent to the kubelet "/pods" endpoint
```

Comment:
We'll need to document the new option in both the config.template and the dockerfile's readme